.github/workflows/stale.yml

```diff
@@ -0,0 +1,44 @@
+name: 'Close inactive issues and PRs'
+
+on:
+  schedule:
+    # Daily at 1:30 AM UTC
+    - cron: '30 1 * * *'
+
+jobs:
+  close-issues-and-pull-requests:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          exempt-draft-pr: true
+          exempt-issue-labels: 'keep-open'
+          exempt-pr-labels: 'keep-open'
+
```

Comment:
```suggestion
          exempt-pr-labels: 'keep-open'

          labels-to-add-when-unstale: 'unstale'
          labels-to-remove-when-stale: 'unstale'

```

Reply:
lgtm, pending discussion in #9435

Reply:
actually, I'll just apply it. I like this direction :-)